question12.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+"""
+Write a NumPy program to remove single-dimensional entries from a specified shape
+"""
+print("specified shape: ", (3, 1, 4))
+print(np.squeeze(np.zeros((3, 1, 4))).shape)
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline